src/Mono.Android/Test/Java.Interop/JnienvTest.cs

```diff
@@ -388,13 +388,18 @@ public void JavaToManagedTypeMapping ()
 		[DllImport ("__Internal")]
 		static extern IntPtr monodroid_typemap_managed_to_java (string java);
 
+		string GetTypeName (Type type)
+		{
+			return type.FullName + ", " + type.Assembly.GetName ().Name;
+		}
+
 		[Test]
 		public void ManagedToJavaTypeMapping ()
 		{
-			var m = monodroid_typemap_managed_to_java (typeof (Activity).AssemblyQualifiedName);
-			Assert.AreNotEqual (IntPtr.Zero, m);
-			m = monodroid_typemap_managed_to_java (typeof (JnienvTest).AssemblyQualifiedName);
-			Assert.AreEqual (IntPtr.Zero, m);
+			var m = monodroid_typemap_managed_to_java (GetTypeName (typeof (Activity)));
+			Assert.AreNotEqual (IntPtr.Zero, m, "`Activity` subclasses Java.Lang.Object, it should be in the typemap!");
+			m = monodroid_typemap_managed_to_java (GetTypeName (typeof (JnienvTest));
```

Comment:
You're missing a closing `)`:

```
Java.Interop/JnienvTest.cs(401,76): error CS1026: ) expected
```

This prevents it [from building](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/2278/console).